### PR TITLE
Updates processor.py to accept class names from config

### PR DESCRIPTION
**Pull request by nicolexirakia**

### CODE REVIEW

File: vsdkx/addon/zoning/processor.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def post_process(self, addon_object: AddonObject) -> AddonObject:
         zone_count[rest_zone_str] = rest_zone_dict
         inference.extra["zoning"] = zone_count
         addon_object.inference = inference
-
+        print(f'Zoning processor results {inference.extra["zoning"]}')
```
Comment (a-ghias-natix) [inline vsdkx/addon/zoning/processor.py:173]: Logging should be done by logging package of python so that we can have more control on the level of logs.

**Review by a-ghias-natix — CHANGES_REQUESTED** [review]

File: vsdkx/addon/zoning/processor.py (hunk 1 above)
Comment (gujjs) [reply vsdkx/addon/zoning/processor.py:173]: we have some bug there, and logging is not working for unknown reason :)

Comment (nicolexirakia) [reply vsdkx/addon/zoning/processor.py:173]: @a-ghias-natix I actually meant to remove this line, I only added it for debug purposes for this PR

Comment (a-ghias-natix) [reply vsdkx/addon/zoning/processor.py]: Ok, then let's remove this line or fix it by using the logging package and @gujjs let's talk on why logging is not working cause it shouldn't have any problem.

**Review by gujjs — APPROVED** [review]

**Pull request merged**